#source/scss/style.scss

```diff
@@ -0,0 +1,98 @@
+@import "nullstyle.scss";
+
+
+ body{
+ 	margin: 0 auto;
+ 	font-family: 'Montserrat', sans-serif;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/chendler_tesla/pullRequest?prid=5633737)